### PR TITLE
Made the "Create Room" button show the create room modal and also added a GetScheduledRooms to Kebab

### PR DESCRIPTION
**Pull request by goldyydev**
Made the "Create Room" button show the create room modal and also added a GetScheduledRooms to Kebab

### CODE REVIEW

**Comment by nadirabbas** [comment]
@overlisted can you please review this, thanks

File: kebab/src/wrapper.ts [hunk 1]
```diff
@@ -14,6 +14,8 @@ export const wrap = (connection: Connection) => ({
   query: {
     getTopPublicRooms: (cursor = 0): Promise<GetTopPublicRoomsResponse> =>
       connection.fetch("get_top_public_rooms", { cursor }),
+    getScheduledRooms: (cursor = "", getOnlyMyScheduledRooms = false): Promise<GetScheduledRoomsResponse> =>
```
Comment (overlisted) [inline kebab/src/wrapper.ts:17]: the type of `cursor` is probably `number | ""`

File: kebab/src/wrapper.ts [hunk 2]
```diff
@@ -14,6 +14,8 @@ export const wrap = (connection: Connection) => ({
...
+      connection.fetch("get_scheduled_rooms", { cursor, getOnlyMyScheduledRooms })
```
Comment (overlisted) [inline kebab/src/wrapper.ts:18]: add a `,` to the end

File: kibbeh/src/modules/dashboard/DashboardPage.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React, { useContext } from "react";
+import { useWrappedConn } from "../../shared-hooks/useConn";
```
Comment (overlisted) [inline kibbeh/src/modules/dashboard/DashboardPage.tsx:2]: where do you use this

**Review by overlisted — CHANGES_REQUESTED** [review]
nothing very critical

File: kebab/src/wrapper.ts (hunk 2 above)
Comment (goldyydev) [reply kebab/src/wrapper.ts]: Darn ESLint

File: kebab/src/wrapper.ts (hunk 1 above)
Comment (goldyydev) [reply kebab/src/wrapper.ts]: Fixed

File: kibbeh/src/modules/dashboard/DashboardPage.tsx (hunk 3 above)
Comment (goldyydev) [reply kibbeh/src/modules/dashboard/DashboardPage.tsx]: Huh, thought I deleted that leftover

**Comment by goldyydev** [comment]
Fixed, fixed and fixed.

File: kibbeh/src/modules/dashboard/FeedController.tsx [hunk 4]
```diff
@@ -15,18 +17,23 @@ export const FeedController: React.FC<FeedControllerProps> = ({}) => {
     refetchOnMount: "always",
     refetchInterval: 10000,
   });
+  const setRoomModal = useShowRoomModal((state) => state.set);
+  const roomModal = useShowRoomModal((state) => state.state);
```
Comment (benawad) [inline kibbeh/src/modules/dashboard/FeedController.tsx:21]: we don't need zustand for this, just do `useState` and boolean true/false

**Review by benawad — CHANGES_REQUESTED** [review]
we don't need zustand for this, just do useState and boolean true/false

**Comment by goldyydev** [comment]
Fixed my tiny brain fart.

**Pull request merged**